test/functionalTest/cases/0000_version_operation/version_via_rest.test

```diff
@@ -40,16 +40,7 @@ Content-Type: application/json
 Fiware-Correlator: REGEX([0-9a-f\-]{36})
 Date: REGEX(.*)
 
-{
-    "orion": {
-        "compile_time": "REGEX(.*)",
-        "compiled_by": "REGEX(.*)",
-        "compiled_in": "REGEX(.*)",
-        "git_hash": "REGEX(([0-9a-f]{40}|nogitversion))",
-        "uptime": "REGEX(.*)",
-        "version": "REGEX(.*)"
-    }
-}
+REGEX({ "orion" : { "version" : "(.*)", "uptime" : "(.*)", "git_hash" : "([0-9a-f]{40}|nogitversion)", "compile_time" : "(.*)", "compiled_by" : "(.*)", "compiled_in" : "(.*)", "release_date" : "(.*)", "doc" : "(.*)" } })
```

Comment:
I don't like too much how this looks like... I would have expected a multi-line output with unordered keys. The `--noPayloadCheck` switch doesn't work exactly as I'd like.

Let me think a bit about possible ways of improving this. Maybe an sligth modification in testHarness.sh could do the trick. @kzangeli do you have any valuable feedback with this regards?

Reply:
What `--noPayloadCheck` does it that is doesn't pass the json output to the beautifier (that also complains in case of JSON errors), that's why the output comes in a single line.
But, what is the reason behind *not* using payload-check?
I can't see any reason ...

Reply:
In this case we want to check the order of the fields, in order to check the new release_date and doc come at the end (see discussion thread in parent issue #2970 for details).

The surprising thing is that, if `--noPayloadCheck` is not passing the JSON to the beautifier then I expect to see the direct JSON, i.e. a JSON with newlines as returned by for instance `curl localhost:1026/version`. 

Some "flattering" effect at orionCurl function is causing that. And maybe having that "flattering" is what we wanted in the pass... but, looking to the modifications in this .test, I cannot remember why :)

Reply:
I guess /version is treated as V2 ?
If not, I have no explanation for the 'single line output'.

Reply:
@fgalan Do we have a consensus on this? I have addressed the rest of the comments but I am waiting for a resolution on this one to continue with the testing.

Reply:
I was talking with @kzangeli about this yesterday. A fix in testHarness.sh to allow multi-line output when `--noPayloadCheck` is enabled in on the way. So I'd suggest to follow the following procedure:

1. Wait for the fix. That will a new PR, from @kzangeli 
2. Solve that PR, eventually merging it into master
3. Fetch changes from `McMutton:master` into `telefonicaid:master`
4. Adjust present PR

Reply:
Ok thank you, I will be waiting for the fix.

Reply:
The PR with the fix by @kzangeli has been merged in master (https://github.com/telefonicaid/fiware-orion/pull/2981). Thus, @McMutton you can go ahead with steps 3 and 4, please.

Reply:
Ok, I am going to start working on it now, I am planning to push the remaining changes on September 26th Tuesday latest.

Reply:
Fixed in e88551b